posthog/session_recordings/session_recording_api.py

```diff
@@ -652,8 +656,98 @@ def destroy(self, request: request.Request, *args: Any, **kwargs: Any) -> Respon
         recording.deleted = True
         recording.save()
 
+        # Also need to remove from playlist items if it's in one
+        SessionRecordingPlaylistItem.objects.filter(playlist__team=self.team, recording=recording).update(deleted=True)
+
```

Comment:
the deleted field is marked as deprecated in the `SessionRecordingPlaylistItem` model

(we can't really ever delete model fields so end up deprecating them and keeping the forever)

(do we not|should we) filter this on the read side?